x/ecocredit/spec/01_concepts.md

```diff
@@ -53,6 +53,22 @@ Tradable credits are credits that can be transferred by the owner to another acc
 
 Retired credits are credits that cannot be transferred between accounts nor can they be unretired. Retired credits are equivalent to burned tokens with the exception that retired credits are actively tracked after being retired. Retiring a credit implies that the holder of a credit is “claiming” the credit as an offset. Credits can be retired upon issuance, retired upon transfer, and retired by the owner of the credits. The retirement location is required upon retirement.
 
+## Order Book
```

Comment:
Maybe let's rename to: "Simple Order Book", and below in description highlight that:
* there is no automatic matching
* users have to manually take the orders

Reply:
Great. I can add the highlights. Maybe "One-Sided Orderbook" to be more descriptive?